omics_orientation/omics_orientation.md

```diff
@@ -0,0 +1,195 @@
+
+### Why study omics?
+
+While studying single molecules or genetic loci is often valuable, biological processes and diseases are generally complicated and are rarely explained by variation in just one component of a system. For example, some genetic diseases and disorders are polygenic (due at least in part to contributions from multiple genes), and many are believed to result from changes in gene regulation rather than changes in the genes themselves. Furthermore, these variations contribute to individual phenotypes in various ways; genetic background and environment play their own roles! In short, there are a lot of moving parts, and studying omics can give researchers a "global" view of biology. This can help explain such things as why some variants cause disease in some individuals and not others, or how the components of a biological system work together to form a whole.
+
+### Quiz: What is omics?
+
+True or False: Omics gives you a narrow, simplified view of biological systems.
+
+[( )] TRUE
+[(X)] FALSE
+***
+<div class = "answer">
+While there is often value in simplifying the study of biological phenomena to one or a few specific targets, omics fields don't take this approach; rather, omics fields leverage the power of big data to try to capture the complexity of biological systems and processes.  
+</div>
+***
+
+## Omics topics
+
+There are many "omics" topics that correspond to various biological domains. The following sections will discuss some of the most popular topics: genomics, transcriptomics, and metabolomics.
+
+### Genomics
+
+As the name suggests, genomics is the study of **genomes**. A genome is the complete genetic information of an organism, and so the study of genomics is generally focused on identifying and studying the effects of variants within this genetic information. In medicine, this can lead to a better understanding of variation in response to pharmacological agents or disease prognoses. A commonly conducted type of genomics study is called a **genome-wide association study** (GWAS) which, according to the National Human Genome Research Institute, is "a research approach used to identify genomic variants that are statistically associated with a risk for a disease or a particular trait" (https://www.genome.gov/genetics-glossary/Genome-Wide-Association-Studies#:~:text=A%20genome%2Dwide%20association%20study,disease%20or%20a%20particular%20trait). In these studies, "thousands of individuals are genotyped for more than a million genetic markers, and statistically significant differences in minor allele frequencies between cases and controls are considered evidence of association" (Hasin et al., 2017).
```

Comment:
```suggestion
As the name suggests, genomics is the study of **genomes**. A genome is the complete genetic information of an organism, and so the study of genomics is generally focused on identifying and studying the effects of variants within this genetic information. In medicine, this can lead to a better understanding of variation in response to pharmacological agents or disease prognoses. A commonly conducted type of genomics study is called a **genome-wide association study** (GWAS) which, [according to the National Human Genome Research Institute](https://www.genome.gov/genetics-glossary/Genome-Wide-Association-Studies#:~:text=A%20genome%2Dwide%20association%20study,disease%20or%20a%20particular%20trait), is "a research approach used to identify genomic variants that are statistically associated with a risk for a disease or a particular trait."  In these studies, "thousands of individuals are genotyped for more than a million genetic markers, and statistically significant differences in minor allele frequencies between cases and controls are considered evidence of association" (Hasin et al., 2017).
```